src/adapters/settings/localstorage.js

```diff
@@ -54,16 +54,24 @@ const localStorageSettings = core => ({
     return Promise.resolve(true);
   },
 
-  load: () => Promise.resolve(Object.keys(localStorage).reduce((o, v) => {
-    let value = localStorage.getItem(v);
-    try {
-      value = JSON.parse(value);
-    } catch (e) {
-      logger.warn('localStorageAdapter parse error', e);
-    }
+  load: () => {
+    const prefix = core.config('settings.prefix');
+
+    const entries = Object
+      .entries(localStorage)
```

Comment:
Not sure if it make any difference but perhaps it's better to do Object.keys here? Since we don't really want to read the values that does not pass the filter.

Reply:
The performance impact in this context would probably be negligible (the real hotspot is when `JSON.parse` is called) in this context, but it's a good suggestion. Fixed :)